packages/ncids-css/package.json

```diff
@@ -42,7 +42,7 @@
 		"postcss": "~8.4.33",
 		"postcss-loader": "6.1.1",
 		"rimraf": "~3.0.2",
-		"sass": "~1.49.0",
+		"sass": "~1.69.0",
```

Comment:
We need to also set a peerDependency since you actually need to have 1.69 for this to work.